pincer/exceptions.py

```diff
@@ -133,6 +133,22 @@ def from_desc(cls, _type: str, max_size: int, cur_size: int):
         )
 
 
+class TaskAlreadyRunning(PincerError):
```

Comment:
Can you make a `TaskError` parent class that will inherit from `PincerError` and will be inherited by all the related error you added? Could be useful. Maybe could even have a attribute that link the the task that cause the error?

Reply:
That's a good idea actually 🤔 
I think I'll add it inside the `TaskError`class. However half errors are raised before the task object is created, so it might be None.

Reply:
maybe just the task function and decorator argument then? feel free to add what seems the best for you

Reply:
Both the error and the traceback are straight-forward when a `TaskIsNotCoroutine` or a `TaskInvalidDelay` error is raised. Including more information is not really relevant.